addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -91,6 +91,10 @@ const RatingPopupComposer = publicWidget.Widget.extend({
                 _t('Review') : this.options.default_message_id ?
                 _t('Edit Review') : _t('Add Review')
             );
+            const reviewButton = document.querySelector(".o_portal_chatter_composer_btn");
+            if (reviewButton) {
+                reviewButton.textContent = this.options.default_message_id ? _t('Update review') : _t('Post review');
+            }
```

Comment:
This seems a bit dirty, can't we instead support a new option inside the "PortalComposer" widget that would be something like "sendButtonLabel"? Would be optional and default to "Send" but allow a bit more customization.